README.md

```diff
@@ -1 +1,13 @@
-# simon-says
\ No newline at end of file
+# simon-says
+
+## Order
+
+1. Build the html
+2. Make the buttons light up when you click them
+3. Play a hardcoded pattern like ['red', 'green', 'blue']
+4. Handle win and lose for hardcoded pattern and user input
+5. Generate a random pattern
+6. Block user input during pattern play
+7. Tie callback into loop so a new pattern is generated when you complete one
+8. Create reset game function
```

Comment:
I actually think this is something that we can make optional. I feel like it's sufficient for them to check for whether it's Game Over or Not. If they finish the main part, and want to write some code to have the game reset, we can make that a bonus.